examples/kubectl-container/Makefile

```diff
@@ -0,0 +1,25 @@
+# Please use like below, to avoid stomping on someone else's (or worse, a
+# production) container. Note the step where you manually copy the GitVersion
+# to use as the TAG.
+#
+# $ make kubectl
+# KUBE_STATIC_OVERRIDES="kubectl" ../../../hack/build-go.sh cmd/kubectl; cp ../../../_output/local/bin/linux/amd64/kubectl .
+# +++ [0602 18:18:53] Building go targets for linux/amd64:
+#     cmd/kubectl
+# +++ [0602 18:18:53] Placing binaries
+# ./kubectl version -c
+# Client Version: version.Info{Major:"0", Minor:"18+", GitVersion:"v0.18.0-243-gc14ab32993e144-dirty", GitCommit:"c14ab32993e144cc81163ea0f6549554f287e12c", GitTreeState:"dirty"}
+# $ make push TAG=v0.18.0-243-gc14ab32993e144-dirty
```

Comment:
Change this line to:
`make push TAG="$(grep -oE 'GitVersion:"[^"]*"' | cut -f 2 -d '"')"`
so that nobody is tempted to cut and paste the constant from your example.